lib/vmpooler/api/v1.rb

```diff
@@ -506,6 +515,15 @@ def validate_date_str(date_str)
                 status 200
                 result['ok'] = true
               end
+            when 'tags'
+              if arg.is_a?(Hash)
```

Comment:
What if `arg` is not a hash? Should this set a status of `400` (bad request) and bail? It is possible - because of the `.each` - that a request with both `lifetime` and `tags` can return a `200 OK`, but actually not record the tags or lifetime, depending on the order of arguments and how they are parsed in the `each` block.